bundles/org.eclipse.jface/src/org/eclipse/jface/dialogs/DialogSettings.java

```diff
@@ -88,6 +90,12 @@ public class DialogSettings implements IDialogSettings {
 	// A Map with all the keys mapped to array of strings.
 	private Map<String, String[]> arrayItems;
 
+	private Deque<String> filterHistory;
+
+	private static final int FILTER_HISTORY_SIZE = 5;
+
+	private static final String EMPTY_SEARCH = ""; //$NON-NLS-1$
```

Comment:
No need for this one. Use `string.isBlank()`

Reply:
done